Resources/Prototypes/Floof/Entities/Objects/Specific/Robotics/Borgmodules.yml

```diff
@@ -0,0 +1,40 @@
+- type: entity
+  id: BaseBorgModuleSecurity
+  parent: BaseBorgModule
+  abstract: true
+  components:
+  - type: Tag
+    tags:
+    - BorgModuleSecurity
+
+- type: entity
+  id: BorgModuleStun
+  parent: [ BaseBorgModuleSecurity, BaseProviderBorgModule ]
+  name: Disabler cyborg module
+  components:
+  - type: Sprite
+    sprite: Floof/Objects/Specific/Robotics/borgmodule.rsi
+    layers:
+    - state: security
+    - state: icon-stun
+  - type: ItemBorgModule
+    items:
+    - WeaponBorgDisabler
+    - WeaponborgTaser
```

Comment:
```suggestion
```

This is to be removed, Taser is a overpowered stun and reason why its is not given to security.
Its is a 1 Shot stun.